pkg/openthread/contrib/openthread.c

```diff
@@ -30,17 +30,30 @@
 #include "at86rf2xx_params.h"
 #endif
 
+#ifdef MODULE_KW41ZRF
+#include "vendor/MKW41Z4.h"
```

Comment:
This vendor file shouldn't be necessary. The `kw41zrf_setup` function and the descriptors should be present in the `kw41zrf.h`